lib/src/specs/enum.dart

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
```

Comment:
[nit] 2020

Reply:
Okay, I have updated the copyright date to 2020